Build/Shared.build.props

```diff
@@ -49,6 +49,8 @@
 
   <Import Project="$(ProjectFeaturesRoot)UseWebForms.props" Condition="'$(UseWebForms)' == 'true'" />
 
+  <Import Project="$(ProjectTypesRoot)Analyzer.props" Condition="'$(ProductType)' == 'Analyzer'" />
+
```

Comment:
revert -> should work already just by adding the `Analyzer.props`/`Analyzer.targets`